### DISCUSSION TITLE
增加新版网页的视频时长选择

### DISCUSSION THREAD

**Discussion by NeilYu9928** [the1812/Bilibili-Evolved#4637] [功能建议 / Ideas]
希望可以添加推荐视频的时长选择，比如可以只允许时长超过600s的视频出现在推荐中

**Comment by Geeyun-JY3** [comment]
就作者两天前发的不会做的功能：https://github.com/the1812/Bilibili-Evolved/discussions/1301#discussioncomment-8787140
含「屏蔽 / 筛选满足某些条件的视频」，感觉这功能无望了。
